bigtable/unit_tests/test_retry.py

```diff
@@ -0,0 +1,54 @@
+import unittest
```

Comment:
I think this should be a system test, not a unit test, since there's an outside dependency.